Modelica/Utilities/Files.mo

```diff
@@ -714,7 +714,7 @@ this might require copying the resource to a temporary folder and referencing th
 
 <p>
 The implementation of this function is tool specific. However, at least Modelica URIs
-(see \"chapter 13.2.3 External Resources\" of the Modelica Specification),
+(see \"Chapter 13.5 External Resources\" of the Modelica Specification),
```

Comment:
```suggestion
(see <a href=\"https://specification.modelica.org/maint/3.6/packages.html#external-resources\">Section&nbsp;13.5 <em>External Resources</em> of the Modelica&nbsp;3.6 specification</a>),
```

We can link directly - as already done in various other locations.

Reply:
Cool! Go for it!